frontend/server/src/Controllers/Contest.php

```diff
@@ -113,6 +115,15 @@ public static function apiList(\OmegaUp\Request $r): array {
         $activeContests = isset($r['active'])
             ? \OmegaUp\DAO\Enum\ActiveStatus::getIntValue(intval($r['active']))
             : \OmegaUp\DAO\Enum\ActiveStatus::ALL;
+        if (isset($r['tab'])) {
```

Comment:
```suggestion
        if (!is_null($tab)) {
```

uno de estos días voy a eliminar el acceso directo a `$r`.